parser.go

```diff
@@ -79,6 +79,11 @@ func (p *Parser) GameState() *GameState {
 	return &p.gameState
 }
 
+// Entities returns the available entities
+func (p *Parser) Entities() *map[int]*st.Entity {
```

Comment:
This should return a `map[int]*st.Entity`
Go maps are already references by default

Reply:
Ooops, my bad. Changed it.